package.json

```diff
@@ -65,4 +65,4 @@
       "pre-commit": "npm run prettier && eslint . --fix && npm run eslint"
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
I don't what made this to change. Also, the version was bumped to `0.6.4` in `package-lock.json`.

Reply:
yeah that's normal. Its fine